### PR TITLE
Splitted timer1 isr

### PR DESCRIPTION
**Pull request by damosvil**
Added DecodeDshot2bit macro md file

### CODE REVIEW

File: _pages/OutsideRange.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: OutsizeRange MACRO
```
Comment (stylesuxx) [inline _pages/OutsideRange.md]: Typo - also I thing he macros are snake cased and not camel cased. Probaly we should keep it consistent with the code.

File: _pages/OutsideRange.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+permalink: /outside-range/
+exclude: true
+---
+
+![OutsideRange]({{'/puml/svgs/DetailedDesign-OutsideRange.svg' | relative_url}})
```
Comment (stylesuxx) [inline _pages/OutsideRange.md]: Please wrap the image in a link to itself, this way we can click on it and have it full - size, or browser filling for that matter.

File: _pages/DecodeDshot2bit.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: DecodeDshot2bit MACRO
+permalink: /decode-dshot-2bit/
+exclude: true
+---
+
+![DecodeDshot2bit]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})
```
Comment (stylesuxx) [inline _pages/DecodeDshot2bit.md]: Please link the image to itself.

File: _pages/DecodeDshot2bit.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: DecodeDshot2bit MACRO
```
Comment (stylesuxx) [inline _pages/DecodeDshot2bit.md]: Snake case instead of camel case.

File: detaileddesing.md [hunk 5]
```diff
@@ -4,6 +4,10 @@ title: Detailed Design
 permalink: /detailed-design/
 ---
 
+## MACROS
+* [DecodeDshot2bit.md]({{ '/decode-dshot-2bit/' | relative_url }})
```
Comment (stylesuxx) [inline detaileddesing.md]: No need to attach ".md" to the link text.

**Review by stylesuxx — CHANGES_REQUESTED** [review]

File: _pages/OutsideRange.md (hunk 1 above)
Comment (damosvil) [reply _pages/OutsideRange.md]: Fixed

File: _pages/DecodeDshot2bit.md (hunk 4 above)
Comment (damosvil) [reply _pages/DecodeDshot2bit.md]: Fixed

File: detaileddesing.md (hunk 5 above)
Comment (damosvil) [reply detaileddesing.md]: Fixed

File: _pages/OutsideRange.md (hunk 2 above)
Comment (damosvil) [reply _pages/OutsideRange.md]: done

File: _pages/DecodeDshot2bit.md (hunk 3 above)
Comment (damosvil) [reply _pages/DecodeDshot2bit.md]: done

File: _pages/DecodeDshot2bit.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: Decode_Dshot_2bit MACRO
+permalink: /decode-dshot-2bit/
+exclude: true
+---
+
+[![Decode_Dshot_2bit]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})]
```
Comment (stylesuxx) [inline _pages/DecodeDshot2bit.md]: This is how the link should look like - sorry, I should have been more clear the first time around.
```
[![Decode_Dshot_2bit]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})
```

File: _pages/OutsideRange.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: Outside_Range MACRO
+permalink: /outside-range/
+exclude: true
+---
+
+[![Outside_Range]({{'/puml/svgs/DetailedDesign-OutsideRange.svg' | relative_url}})]
```
Comment (stylesuxx) [inline _pages/OutsideRange.md]:
```
[![Outside_Range]({{'/puml/svgs/DetailedDesign-OutsideRange.svg' | relative_url}})]({{'/puml/svgs/DetailedDesign-OutsideRange.svg' | relative_url}})
```

**Review by stylesuxx — CHANGES_REQUESTED** [review]

File: _pages/DecodeDshot2bit.md (hunk 6 above)
Comment (damosvil) [reply _pages/DecodeDshot2bit.md]: fixed

File: _pages/OutsideRange.md (hunk 7 above)
Comment (damosvil) [reply _pages/OutsideRange.md]: fixed

**Review by stylesuxx — APPROVED** [review]

**Pull request merged**